core/blockchain_ext_test.go

```diff
@@ -1488,7 +1489,7 @@ func InsertChainInvalidBlockFeeTest(t *testing.T, create createFunc) {
 	})
 	require.NoError(err)
 	_, err = blockchain.InsertChain(chain)
-	require.ErrorIs(err, dummy.ErrInsufficientBlockGas)
+	require.ErrorIs(err, customheader.ErrInsufficientBlockGas)
```

Comment:
For my own understanding, how is it that `ModeSkipBlockFee: true` above in this test, but then we are still doing performing the block fee verification? I must be missing something...